main/src/ca/uwaterloo/flix/api/lsp/provider/completion/BuiltinTypeCompleter.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 Paul Butcher
```

Comment:
You should always add your name to files that you touch :)

Reply:
Add or replace? Also when I'm still just kinda refactoring?

Reply:
Just add, its always okay to add.

Reply:
Preferences in terms of adding? 

Comma-seperation -> Old name, Lukas Rønn

Or newline with year and name?

Reply:
Just name. Add at end.